pom.xml

```diff
@@ -181,7 +181,7 @@
     </build>
 
     <properties>
-        <cas.version>4.1.0-SNAPSHOT</cas.version>
+        <cas.version>4.1.0</cas.version>
```

Comment:
generally, when writing extension type libraries, you would use the lowest supported version. If there are no API changes between 4.1.0 and 4.1.1 that would affect this module, 4.1.0 would be more appropriate. Has there been?